src/collections/news/2023-04-15-Baby-Bliss-Bot-AI-Techniques.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Baby Bliss Bot - Experiments with AI
+slug: clusive-latest-features
+date: "2023-04-15"
+excerpt: Exploring AI Machine Learning models with Bliss symbols.
+---
+A number of different artificial intelligence machine learning systems were
+explored in order to acquire a basic knowledge of the systems, how to train
+them, and what they produce, as well as an idea of their value to the
+[Baby Bliss Bot Project](https://github.com/fluid-project/floeproject.org/blob/eb04bf9b0774f452145376f3495933eb3ce81c10/src/collections/news/2023-04-12-BBB-Intro.md).
```

Comment:
I just merged the intro post. Time to fix the link. :)

Reply:
Done :-)